.taskcluster.yml

```diff
@@ -56,6 +56,59 @@ tasks:
           owner: cdenizet@mozilla.com
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
 
+      - taskId: {$eval: as_slugid("build_documentation")}
+        created: {$fromNow: ''}
+        deadline: {$fromNow: '2 hour'}
+        provisionerId: proj-relman
+        workerType: ci
+        payload:
+          maxRunTime: 3600
+          image: "rust:buster"
+          command:
+            - "/bin/bash"
+            - "-cx"
+            - "git clone --recursive --quiet ${repository} &&
+               cd rust-code-analysis &&
+               git -c advice.detachedHead=false checkout ${head_rev} &&
+               cargo install mdbook --no-default-features --features output --vers \"^0.1.0\" &&
+               cargo doc --release &&
+               cd rust-code-analysis-book &&
+               mdbook build"
+          artifacts:
+            public/book:
+              expires: {$fromNow: '2 weeks'}
+              path: rust-code-analysis-book/book
+              type: file
+        metadata:
+          name: rust-code-analysis documentation
+          description: rust-code-analysis build documentation
+          owner: cdenizet@mozilla.com
+          source: ${repository}/raw/${head_rev}/.taskcluster.yml
+
+      - taskId: {$eval: as_slugid("test_gh_pages")}
```

Comment:
You can remove this task and put its commands in the `deploy_release` task

Reply:
Yep, just testing for now. If it works I'm going to remove that task